torchtune/config/_utils.py

```diff
@@ -173,6 +173,11 @@ def _merge_yaml_and_cli_args(yaml_args: Namespace, cli_args: List[str]) -> DictC
         # key string to reflect this
         if k in yaml_kwargs and _has_component(yaml_kwargs[k]):
             k += "._component_"
+
+        # None passed via CLI will be parsed as string, but we really want OmegaConf null
+        if v == "None":
```

Comment:
Should it be v.lower() == “none”? To avoid the None/none case

Reply:
I might leave it as is.. I know it's not a heavily-used API in the library as of today, but e.g. https://github.com/pytorch/torchtune/blob/f560cbb21567ebf4852267681f7202526fa249ca/torchtune/training/activations.py#L75

Since we already have a case that's using 'none' as a string, I don't wanna mess with that